source/api/permissionGroup.service.ts

```diff
@@ -67,10 +66,10 @@ export class PermissionGroupService {
         ).pipe(map((o: any) => FlexiCoreDecycle.retrocycle(o)));
     }
 
-    public getPermissionGroups(body?: PermissionGroupFiltering, authenticationKey?: string, extraHttpRequestParams?: any, observe?: 'body', reportProgress?: boolean): Observable<PaginationResponse<PermissionGroup>>;
-    public getPermissionGroups(body?: PermissionGroupFiltering, authenticationKey?: string, extraHttpRequestParams?: any, observe?: 'response', reportProgress?: boolean): Observable<HttpResponse<PaginationResponse<PermissionGroup>>>;
-    public getPermissionGroups(body?: PermissionGroupFiltering, authenticationKey?: string, extraHttpRequestParams?: any, observe?: 'events', reportProgress?: boolean): Observable<HttpEvent<PaginationResponse<PermissionGroup>>>;
-    public getPermissionGroups(body?: PermissionGroupFiltering, authenticationKey?: string, extraHttpRequestParams?: any, observe: any = 'body', reportProgress: boolean = false): Observable<any> {
+    public getPermissionGroups(body?: PermissionGroupFilter, authenticationKey?: string, extraHttpRequestParams?: any, observe?: 'body', reportProgress?: boolean): Observable<PaginationResponse<PermissionGroup>>;
```

Comment:
please remove authenticationKey parameter from all apis , this should allways be controlled using an interceptor